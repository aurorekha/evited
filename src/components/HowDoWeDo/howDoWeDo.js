import React from 'react';
import ScrollAnimation from 'react-animate-on-scroll';
import AOS from 'aos';

import 'aos/dist/aos.css';
import './howDoWeDo.scss';

AOS.init();

function HowDoWeDo() {
  return (
    <div className="howDoWeDo">
      <h1>How does this work?</h1>
      <div data-aos="filp-down" className="howDivs">
        <h2>Plan</h2>
        <p>We sit you take details of what you want and when you want</p>
      </div>
      <div data-aos="filp-down" className="howDivs">
        <h2>Design</h2>
        <p>We then sit with a deisgner and come back
          with super neat design which we think will go with what you
          want and with whatyou need. And agree on a deisgn with little tweaks
        if needed</p>
      </div>
      <div data-aos="flip-down" className="howDivs">
        <h2>Develop</h2>
        <p>We develop and bring the designs to life and set you up with a mailbox
        so that you can control your RSVPs.</p>
      </div>
      <div data-aos="flip-down" className="howDivs">
        <h2>Deploy</h2>
        <p>We deploy to a domain that will be easy to share for you to the people
        you want to invite.</p>
      </div>
      <div data-aos="flip-down" className="howDivs">
        <h2>Celebrate</h2>
        <p>Oh come on! Does this really need any explaining?</p>
      </div>
    </div>
  )
}

export default HowDoWeDo;
