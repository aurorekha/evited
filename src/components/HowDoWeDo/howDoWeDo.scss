@font-face {
  font-family: "arapey";
  src: url("../../fonts/arapey/arapey-italic-webfont.woff") format("woff"),
}

.howDoWeDo {
  font-family: "arapey";
  color: white;
  background: #FEB5AE;
  height: 800px;
  text-align: center;
  h1 {
    padding-top: 2rem;
  }
}

.howDivs {
  width: 40%;
  height: 300px;
  float: left;
}
