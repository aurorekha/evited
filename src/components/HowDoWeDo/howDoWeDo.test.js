import React from 'react';
import HowDoWeDo from './howDoWeDo';
import renderer from 'react-test-renderer';

it('renders HowDoWeDo section correctly', () => {
  const tree = renderer.create(
    <HowDoWeDo />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
