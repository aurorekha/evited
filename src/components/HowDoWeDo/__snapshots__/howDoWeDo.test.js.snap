// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HowDoWeDo section correctly 1`] = `
<div
  className="howDoWeDo"
>
  <h1>
    How does this work?
  </h1>
  <div
    className="howDivs"
    data-aos="filp-down"
  >
    <h2>
      Plan
    </h2>
    <p>
      We sit you take details of what you want and when you want
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="filp-down"
  >
    <h2>
      Design
    </h2>
    <p>
      We then sit with a deisgner and come back with super neat design which we think will go with what you want and with whatyou need. And agree on a deisgn with little tweaks if needed
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Develop
    </h2>
    <p>
      We develop and bring the designs to life and set you up with a mailbox so that you can control your RSVPs.
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Deploy
    </h2>
    <p>
      We deploy to a domain that will be easy to share for you to the people you want to invite.
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Recheck
    </h2>
    <p>
      You check if everything is in place as per agreements
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Payment
    </h2>
    <p>
      It's payday for us once we have everything in place and tick the final box.
    </p>
  </div>
</div>
`;

exports[`renders correctly 1`] = `
<div
  className="howDoWeDo"
>
  <h1>
    How does this work?
  </h1>
  <div
    className="howDivs"
    data-aos="filp-down"
  >
    <h2>
      Plan
    </h2>
    <p>
      We sit you take details of what you want and when you want
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="filp-down"
  >
    <h2>
      Design
    </h2>
    <p>
      We then sit with a deisgner and come back with super neat design which we think will go with what you want and with whatyou need. And agree on a deisgn with little tweaks if needed
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Develop
    </h2>
    <p>
      We develop and bring the designs to life and set you up with a mailbox so that you can control your RSVPs.
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Deploy
    </h2>
    <p>
      We deploy to a domain that will be easy to share for you to the people you want to invite.
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Recheck
    </h2>
    <p>
      You check if everything is in place as per agreements
    </p>
  </div>
  <div
    className="howDivs"
    data-aos="flip-down"
  >
    <h2>
      Payment
    </h2>
    <p>
      It's payday for us once we have everything in place and tick the final box.
    </p>
  </div>
</div>
`;
